### PR TITLE
#167290437 Clear out warring report by test

### PR DESCRIPTION
**Pull request by blackshady**
#### What does this PR do? 
 Eradicate the warrings showing up while running test.

#### Description of Task to be completed?

- [x] Clear out warnings form the console
- [x]  Add some test case

#### How should this be manually tested?

1.  Pull this branch `git pull ch-fix-warning-report-167290437`
2.  Run `yarn or yarn install`
3.  After you dependencies are done installing run `yarn test`

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#167290437](https://www.pivotaltracker.com/story/show/167290437)

#### Postman Documentation
- N/A

#### Screenshots (If applicable)
Screenshot 2019-07-17 at 3 39 29 PM

### CODE REVIEW

**Review by bozicschucky — APPROVED** [review]
I have tested the branch locally and everything seems to work as expected, good work @blackshady

**Review by 09arnold — COMMENTED** [review]
Looks good, but there's some merge conflicts and code climate issues you should look into

**Review by akhilome — APPROVED** [review]
This looks good to go. You do have multiple commits, though.

File: src/components/developerCards/index.jsx [hunk 1]
```diff
@@ -83,26 +82,13 @@ class DeveloperCard extends Component {
       const name = card.fellowName;
       const newName = _.split(name, ',');
       const firstInitials = newName[0].charAt(0);
-      const secondInitials = newName[1] && newName[1].charAt(1);
+      const secondInitials = newName[1].charAt(1);
```
Comment (ssewilliam) [inline src/components/developerCards/index.jsx]: @blackshady   This change is likely to return the bug that was solved [here](https://www.pivotaltracker.com/story/show/167191688) and fixed through this [PR](https://github.com/andela/bp-esa-frontend/pull/101)

File: src/components/developerCards/index.jsx [hunk 2]
```diff
@@ -83,26 +82,13 @@ class DeveloperCard extends Component {
...
-                <div className="tooltip-container" id="offboarding-info-icon">
-                  <img className="info-icon onBoarding-icon" src={offboarding} alt="offboarding icon" />
-                  <span className="tooltiptext">Off-boarding</span>
-                </div>
-              )}
-            <div id="more-info-icon" className="tooltip-container" onClick={() => { openModal(); changeModalTypes(card); }}>
-              <img className="info-icon" src={InfoIcon} alt="info icon" role="presentation" id={`${index}-id`} />
-              <span className="tooltiptext">Details</span>
-            </div>
+            { /* TODO Update the onboarding icon to match mockups */}
```
Comment (ssewilliam) [inline src/components/developerCards/index.jsx]: @blackshady were you expecting to do this in this PR and forgot about it or I should create a story about it.

File: src/components/ReportPage/index.jsx [hunk 3]
```diff
@@ -459,8 +456,8 @@ export const mapDispatchToProps = dispatch => ({
   fetchAllAutomation: (pagination, filters) => dispatch(fetchAutomation(pagination, filters)),
   fetchUpdates: () => dispatch(fetchRealTimeReport()),
   resetUpdates: () => dispatch(resetRealTimeReport()),
-  fetchStat: period => dispatch(fetchStatsRequest(period)),
-  retryFailedAutomation: automationId => dispatch(retryAutomation(automationId)),
+  fetchStat: (period) => dispatch(fetchStatsRequest(period)),
+  retryFailedAutomation: () => dispatch(retryAutomation()),
```
Comment (ssewilliam) [inline src/components/ReportPage/index.jsx]: Hey, @blackshady  retryAutomations expects AutomationId but you have taken it out. Is this intended?

https://github.com/andela/bp-esa-frontend/blob/28fb2fc768824f0bd129ef380e008548153da6b1/src/redux/actions/automationActions.js#L27

File: src/components/developerCards/index.jsx (hunk 1 above)
Comment (blackshady) [reply src/components/developerCards/index.jsx]: Hi @ssewilliam this was coming from a different commit. it has been resolved.

**Review by ssewilliam — COMMENTED** [review]
Hey @blackshady please resolve the merge conflicts

**Comment by blackshady** [comment]
Hi @ssewilliam the merge conflict has been fixed

**Pull request merged**